_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -268,7 +268,7 @@ Element has ARIA role, but no ARIA states or properties
 
 `aria-checked` state with empty value
 
-**Note**: The HTML validator flags an `aria-checked` attribute with an empty value as an issue. However WAI-ARIA 1.1 indicates `aria-checked` has a default value of `undefined`.
+**Note**: The HTML validator flags an `aria-checked` attribute with an empty value as an issue. However, since WAI-ARIA 1.1, `aria-checked` has a default value of `undefined`.
```

Comment:
```suggestion
**Note**: The HTML validator flags an `aria-checked` attribute with an empty value as an issue. However, since WAI-ARIA 1.2, `aria-checked` has a default value of `undefined`.
```

Reply:
With the "since", I'd say "Since 1.1" is still the correct version 🤔 Maybe
```suggestion
**Note**: The HTML validator flags an `aria-checked` attribute with an empty value as an issue. However, in WAI-ARIA 1.2, `aria-checked` has a default value of `undefined`.
```